apps/withdraw-pool/src/utils/math.ts

```diff
@@ -6,3 +6,10 @@ export function multiplyValueByPct(
 ): BigNumber {
   return BigNumber.from(value).mul(pct).div(100);
 }
+
+export function getPctFromValue(
+  value: BigNumberish,
+  total: BigNumberish
+): number {
+  return BigNumber.from(value).mul(100).div(total).toNumber();
```

Comment:
a bit picky but if `total` * 100 is larger than value this will always return 0, right? So:
```
CASE value * 100 = total
const value = 1
const total = 100
getPctFromValue(value, total) => 1.0

CASE value * 100 < total
const value = 1
const total = 1000
getPctFromValue(value, total) => 0
```

could that be an issue or have I got this wrong perhaps?